homeassistant/components/zha/entity.py

```diff
@@ -107,6 +106,9 @@ def async_update_state_attribute(self, key: str, value: Any) -> None:
     def async_set_state(self, attr_id: int, attr_name: str, value: Any) -> None:
         """Set the entity state."""
 
+    async def async_added_to_hass(self) -> None:
```

Comment:
Is this needed?

Reply:
Oh I think it’s to override it?

will need to wait until I’m back at my desk to look at this one

Reply:
The old override is is the empty method of  `Entity`.

Reply:
I think is this is solved now.